src/main/java/de/tum/cit/aet/artemis/communication/web/FaqResource.java

```diff
@@ -166,14 +176,37 @@ public ResponseEntity<Void> deleteFaq(@PathVariable Long faqId, @PathVariable Lo
     @EnforceAtLeastStudent
     public ResponseEntity<Set<FaqDTO>> getFaqForCourse(@PathVariable Long courseId) {
         log.debug("REST request to get all Faqs for the course with id : {}", courseId);
-
-        Course course = courseRepository.findByIdElseThrow(courseId);
-        authCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.STUDENT, course, null);
         Set<Faq> faqs = faqRepository.findAllByCourseId(courseId);
         Set<FaqDTO> faqDTOS = faqs.stream().map(FaqDTO::new).collect(Collectors.toSet());
         return ResponseEntity.ok().body(faqDTOS);
     }
 
+    /**
+     * GET /courses/:courseId/faq-status/:faqState : get all the faqs of a course in the specified status
+     *
+     * @param courseId the courseId of the course for which all faqs should be returned
+     * @param faqState the state of all returned FAQs
+     * @return the ResponseEntity with status 200 (OK) and the list of faqs in body
+     */
+    @GetMapping("courses/{courseId}/faq-state/{faqState}")
+    @EnforceAtLeastStudent
+    public ResponseEntity<Set<FaqDTO>> getAllFaqForCourseByStatus(@PathVariable Long courseId, @PathVariable String faqState) {
```

Comment:
Can't you directly use the enum as parameter here? I think we do this in other endpoints as well